dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -77,6 +73,10 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         final Optional<List<String>> authorizedRepos = authorizedRepos(organization, githubToken);
         response.addHeader(X_TOTAL_COUNT, String.valueOf(lambdaEventDAO.countByOrganization(organization, authorizedRepos)));
         response.addHeader(ACCESS_CONTROL_EXPOSE_HEADERS, X_TOTAL_COUNT);
```

Comment:
Denis recently added these in https://github.com/dockstore/dockstore/pull/5688 to help support pagination so that the total count of events is available in the header of any response of this endpoint and the UI can use that to display the total number of events available.

The [allPublishedWorkflows](https://github.com/dockstore/dockstore/blob/4c12b34709b9a94e74202a75629b7712efbe9c29/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L812) is a similar endpoint that's used by the UI to implement pagination on https://dockstore.org/workflows so you can likely take inspiration from the code for that page. If you open the browser developer tools and go to the Networks tab, you can see how the endpoint is called with each pagination action (ex: user clicks next page, the UI invokes the endpoint with `offset=10` to get the next 10 workflows to display).